jobs/starter/smoke-test/Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env groovy
+
+// A Job to run a smoke test in a cluster
+// 
+// It invokes the 'smoketest' operation of 'cicd-control.sh' for the specified cluster.
+// Optionally emails test results.
+
+node('openshift-build-1') {
+
+    properties([
+            [   $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        $class: 'hudson.model.StringParameterDefinition',
+                        name: 'CLUSTER_NAME',
+                        description: 'Cluster to run tests on',
+                    ],
+                    [
+                        $class: 'hudson.model.BooleanParameterDefinition',
+                        defaultValue: false,
+                        description: 'Send email notifications',
+                        name: 'MAIL_RESULTS'
+                    ],
+                    [
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: 'aos-devel@redhat.com',
+                        description: 'Success Mailing List',
+                        name: 'MAIL_LIST_SUCCESS'
+                    ],
+                    [
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: 'aos-qe@redhat.com',
+                        description: 'Failure Mailing List',
+                        name: 'MAIL_LIST_FAILURE'
+                    ],
+                ]
+            ],
+        ]
+    )
+
+    def deploylib = load( "pipeline-scripts/deploylib.groovy" )
+    deploylib.initialize()
```

Comment:
Cluster jobs have changed a bit while you have worked this card.

Populating cluster names:
https://github.com/openshift/aos-cd-jobs/blob/master/jobs/starter/perf/Jenkinsfile#L14-L17
https://github.com/openshift/aos-cd-jobs/blob/master/jobs/starter/perf/Jenkinsfile#L28

Initializing deploylib:
https://github.com/openshift/aos-cd-jobs/blob/master/jobs/starter/perf/Jenkinsfile#L39

Setting up ssh agent (CLUSTER_ENV instead of _NAME): 
https://github.com/openshift/aos-cd-jobs/blob/master/jobs/starter/perf/Jenkinsfile#L51

Reply:
Sorry I missed these, will update...

Reply:
You didn't miss them, they changed underneath you :)